Makefile

```diff
@@ -61,7 +61,7 @@ install-dev-egg: $(SITE_PACKAGES)/c2corg_api.egg-link
 
 .PHONY: serve
 serve: install development.ini
-	.build/venv/bin/pserve --reload development.ini
+	.build/venv/bin/gunicorn --paste --reload development.ini
```

Comment:
This command is incorrect and cause the following error:
```
alex@c2corgv6-demo:~/v6_api$ .build/venv/bin/gunicorn --paste --reload development.ini
usage: gunicorn [OPTIONS] [APP_MODULE]
gunicorn: error: argument --paste/--paster: expected one argument
```
Using the following parameters order seems to work:
```
.build/venv/bin/gunicorn --paste development.ini --reload
```